_posts/2019-05-09-How-We-Release-Code.md

```diff
@@ -0,0 +1,96 @@
+---
+title: How we release open source projects
+author: Per Ploug
+date: 2019-05-09
+banner:
+  image: connected.jpg
+---
+> Zalando currently have over 200 open source projects on Github and each month there are approximately 2 new projects proposed for publishing as open source.
+> Over the years we've refined our release process to ensure that it is easy and transparent to publish projects as open source and that we keep
+> our maintainers accountable for the well-being of the projects which they oversee.
+
+This blog post is an attempt at describing how we manage the process of proposing, reviewing and approving projects to become open source code while at the same time ensuring that code we release follow our compliance rules and that the maintainers of the projects are aware of the risks and responsibilities involved.
```

Comment:
"""
This blog post describes how we manage the process of proposing, reviewing 
and approving projects to become open source, while at the same time ensuring 
project code follows our compliance rules, and the maintainers of the projects 
are aware of their responsibilities.
"""